docs/DeveloperGuide.md

```diff
@@ -734,6 +734,30 @@ Extensions
     * 3a1. UniGenda shows an error message.
 
   Use case ends.
+  
+    
+**Use case: Find a common timing that contacts who share a similar day are free on a particular day**
+
+**MSS**
+
+1. User requests to list persons.
+2. User chooses to view the free timings of a group of contacts who share a similar tag, on a particular day.
+3. UniGenda shows the timings the contacts with that particular tag is available on the particular day.
+
+**Extensions**
+* 1a. The given tag is not attached to any contact in UniGenda.
+    
+    * 1a1. UniGenda shows an error message.
+  
+  Use case ends.
+
+* 2a. The contacts which share the given tag do not have any events scheduled on the given date.
+
+    * 2a1. UniGenda shows the user a message which indicates that the whole day is free for contacts who share the inputted tag.
+    
```

Comment:
Here as well. Use case ends. Also, Yan Han made some changes to the format of the use cases. Merge them before you continue.